sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Microsoft.Azure.Management.Maintenance.csproj

```diff
@@ -6,10 +6,10 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.Maintenance</PackageId>
     <Description>Microsoft Azure Maintenance Management Library</Description>
-    <Version>1.0.0</Version>
+    <Version>2.0.0</Version>
```

Comment:
We follow sementic versioning, so a major version update means there's going to be breaking changes for the user. Are you sure about this?

In normal case when there's some non-breaking feature addition, we usually update the minor version, i.e. `1.1.0`. If that suits you, don't forget the version in AssemblyInfo.cs

Reply:
I see ok I have updated only the minor version then